test/metric_dimension/metrics_aggregate_dimension_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
```

Comment:
Nit: integration is no longer needed since this is not in the same dir as the agent's unit tests. For we should use !windows since this should work on darwin.

Reply:
yeah but for now everything is assuming tests run with integration flag (terraform, all files) so not something I wanna change in this PR. Am I missing something?

Reply:
I think Seth might have removed that flag. I think when you rebase, we can confirm.

Reply:
ah yes he did